packages/frontend/pages/EthComponents.tsx

```diff
@@ -0,0 +1,13 @@
+import type { NextPage } from "next";
```

Comment:
Let's not use this page. Let's put it in the home page, after the Address component.

IF there are conflicts in other PRs, we'll fix it.